sunpy/net/dataretriever/attrs/gbm.py

```diff
@@ -0,0 +1,15 @@
+from sunpy.net.attr import SimpleAttr
+
+__all__ = ['Detector', 'Datatype']
+
+
+class Detector(SimpleAttr):
```

Comment:
I am pretty sure the vso has a detector attribute, so we shouldn't re-define this here, we should just reuse the VSO one by making it importable from `sunpy.net.attrs`

Reply:
oh somehow missed that - I've updated now to use vso.Detector